app/models/active_operator/operation.rb

```diff
@@ -1,15 +1,20 @@
 class ActiveOperator::Operation < ApplicationRecord
   belongs_to :record, polymorphic: true
 
-  def operator
-    operator_class.new(self)
-  end
-
-  delegate :request, :process, to: :operator
+  def perform
+    save! unless persisted?
```

Comment:
I wonder if we need this. One of my Active Record tricks is using `update!` on a `new_record?` which will still create in the database. You could let this fall through and let `request!` call `update!` and save a DB query.

Reply:
Yeah, there may be a better approach here... I was trying to protect against the case that someone does this: `Location.new.geocoding.perform`, which would cause the request to be made without validating that the Location has an address to be geocoded. It also seemed to good to match the behavior of `perform_later`, but maybe that doesn't matter as much?

Reply:
Another point, I don't know if Active Storage guards against this case.